.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Sherlock Platform
+
+on:
+  # Triggers the workflow on push or pull request events but only for the idea/242.21829.142 branch
+  push:
+    branches: [ idea/242.21829.142 ]
```

Comment:
I feel like we should, at some point, update the name of this branch. (maybe it should be done before this change, or can we put `.*` here? )